### ISSUE TITLE
VNC "Failed to Connect" error

### ISSUE THREAD

**Issue by menos08642** [jlesage/docker-crashplan-pro#425]
Frequently when hitting the IP:port for my running container, I get a red bar at the top with a "Failed to Connect" message. This is resolved by restarting the container. This can have the unintended consequence of really mucking with the running crashplan instance.
I've also noticed that I can fix it by killing the xvnc process running inside the container itself. 
There is nothing in the logs that looks relevant. Ideas on what to look for?

**Comment by jlesage** [comment]
Do you see the problem with different browsers ?

**Comment by menos08642** [comment]
Yes, I've tried with Edge, Chrome and Firefox. I've set CONTAINER_DEBUG to 1 but I'm not seeing any additional logging or indication of what is crashing/hanging. Is there anyway to turn on additional verbosity of the xvnc process?

**Comment by jlesage** [comment]
`CONTAINER_DEBUG` already increases verbosity of xvnc...

Did you see the problem with the latest image ?  In this image, xvnc has been upgraded to the latest version.

**Comment by menos08642** [comment]
The latest version has been up for 4 days now without the xvnc crash. That looks like it may have solved it.

**Comment by jlesage** [comment]
Thanks for the update, closing this issue.
